src/intentManager.js

```diff
@@ -4,15 +4,13 @@ define([
     'vellum/util',
     'underscore',
     'jquery',
-    'tpl!vellum/templates/intent_templates',
     'vellum/core'
 ], function (
     mugs,
     widgets,
     util,
     _,
-    $,
-    intent_templates
+    $
```

Comment:
A brief intro - something explaining how intents can be templated or custom, and...I'm not sure what else should be in the intro, because I haven't finished reading the PR yet - would make this file quite a bit better.